Core/automation/lib/python/configuration.py.example

```diff
@@ -4,4 +4,9 @@ LOG_PREFIX = "jsr223.jython"
 adminEmail = "admin_email@some_domain.com"
 
 openhabHost = "localhost"
-openhabPort = "8080"# "8443"
\ No newline at end of file
+openhabPort = "8080"# "8443"
+
+# Pushover related.
+# Uncomment if you are going to use Pushover notifications https://pushover.net
+# PUSHOVER_PRIO = {'LOWEST': -2, 'LOW': -1, 'NORMAL': 0, 'HIGH': 1, 'EMERGENCY': 2}
+# PUSHOVER_DEF_DEV = "MyCellPhone" # Default device for Pushover messages to be delivered to 
```

Comment:
I think this is the last bit! These should go into a community configuration.py.example.

Reply:
The only reference to it among the community scripts is in the ideAlarm `custom_verbose_example.py`.

So I've moved it to ideAlarm.